### DISCUSSION TITLE
Unable to install older versions of boost.

### DISCUSSION THREAD

**Discussion by anmoldhuria** [microsoft/vcpkg#42142] [Q&A]
Hello everyone, I am trying to install boost-1.68.0.

I was following this https://learn.microsoft.com/en-us/vcpkg/consume/boost-versions. Got the baseline version as explained there.
vcpkg.json I have tried:

```
{
    "name": "boost-vcpkg",
    "version": "1.0.0",
    "dependencies": [
        {
            "name": "boost",
            "version>=": "1.76.0"
        }
    ],
    "builtin-baseline": "761c81d43335a5d5ccc2ec8ad90bd7e2cbba734e",
    "overrides": [
        {
            "name": "boost",
            "version": "1.68.0"
        }
    ]
}
```

The exception I am facing is :
[DEBUG] Failed to open: C:\repos\tmp\vcpkg_buildtrees\versioning_\versions\liblzma\5f491131f98307f42a98db7ad0b5455225481284\vcpkg.json
[DEBUG] Failed to open: C:\repos\tmp\vcpkg_buildtrees\versioning_\versions\zlib\53a4615c8bb9b98a3864b834a6bbe51cc6c849ef\vcpkg.json
error: the baseline does not contain an entry for port boost-disjoint-sets
error: the baseline does not contain an entry for port boost-signals

Also when I am trying baseline-commit as it has 1.68.0 commit. https://github.com/microsoft/vcpkg/commit/eae02c95b33ec0272a232395820379e0a8b2393d

I am getting these errors:

```
error: while checking out baseline from commit 'eae02c95b33ec0272a232395820379e0a8b2393d', failed to `git show` versions/baseline.json. This may be fixed by fetching commits with `git fetch`.
error: git failed with exit code: (128).
fatal: path 'versions/baseline.json' exists on disk, but not in 'eae02c95b33ec0272a232395820379e0a8b2393d'
while checking out baseline eae02c95b33ec0272a232395820379e0a8b2393d
```
Is there a way to install older version of boost ?

**Comment by autoantwort** [comment]
> Also when I am trying baseline-commit as it has 1.68.0 commit. https://github.com/microsoft/vcpkg/commit/eae02c95b33ec0272a232395820379e0a8b2393d

Back then vcpkg did not support versioning. So you have to select a newer version.
